### PR TITLE
spam nvrtc options

### PR DESCRIPTION
**Pull request by jjsjann123**
TORCH_WARN on nvrtc debug option impacting performance.

### CODE REVIEW

File: torch/csrc/jit/codegen/cuda/executor_utils.cpp [hunk 1]
```diff
@@ -957,6 +957,8 @@ std::pair<NvrtcFunction, std::string> nvrtcCompile(
   }
 #else
   if (disable_fma) {
+    TORCH_WARN(
```
Comment (naoyam) [inline torch/csrc/jit/codegen/cuda/executor_utils.cpp]: Would it make more sense to have this warning in `isDisabled()`?

**Review by naoyam — APPROVED** [review]
LGTM

**Comment by csarofeen** [comment]
@jjsjann123 want to merge this in?

File: torch/csrc/jit/codegen/cuda/executor_utils.cpp (hunk 1 above)
Comment (jjsjann123) [reply torch/csrc/jit/codegen/cuda/executor_utils.cpp]: Good idea, that's also future proof in case we are using it somewhere else. :+1:

**Pull request merged**